pytest.ini

```
@@ -1,9 +1,9 @@
 [pytest]
 env =
     AWS_DEFAULT_REGION=eu-west-2
-addopts = --tb=short --numprocesses auto --dist loadgroup
+; addopts = --tb=short --numprocesses auto --dist loadgroup
 ; You may need to use the addopts setting below to unbreak debugging in your IDE
-; addopts = --tb=short
+addopts = --tb=short
```

```suggestion
;addopts = --tb=short
```

unless, for some reason, this was intentional?

---

It's temporary while I rework this subsequent to Friday afternoons end of day conversation. Hold your fire!